lib/libs/email/content/new-submission/emailTemplates/MedSpaState.tsx

```diff
@@ -1,62 +1,59 @@
+import { Text } from "@react-email/components";
+import { styles } from "../../email-styles";
 
-export const MedSpaStateEmail = (props: {
-  variables: OneMac & CommonVariables;
-}) => {
+export const MedSpaStateEmail = (props: { variables: Events["NewMedicaidSubmission"] & CommonEmailVariables }) => {
   const variables = props.variables;
+  const previewText = `Medicaid SPA &${variables.id} Submitted`;
+  const heading = "This response confirms that you submitted a Medicaid SPA to CMS for review:";
   return (
-    <Html lang="en" dir="ltr">
-      <Container>
-        <h3>
-          This response confirms that you submitted a Medicaid SPA to CMS for
-          review:
-        </h3>
-        <PackageDetails
-          details={{
-            "State or territory": variables.territory,
-            Name: variables.submitterName,
-            "Email Address": variables.submitterEmail,
-            "Medicaid SPA ID": variables.id,
-            "Proposed Effective Date": formatDate(
-              variables.notificationMetadata?.proposedEffectiveDate,
-            ),
-            "90th Day Deadline": formatNinetyDaysDate(
-              variables.notificationMetadata?.submissionDate,
-            ),
-            Summary: variables.additionalInformation,
-          }}
-        />
-        <p>
-          This response confirms the receipt of your Medicaid State Plan
-          Amendment (SPA or your response to a SPA Request for Additional
-          Information (RAI)). You can expect a formal response to your submittal
-          to be issued within 90 days, before{" "}
-          {formatNinetyDaysDate(variables.notificationMetadata?.submissionDate)}
-          .
-        </p>
-        <MailboxSPA />
-        <ContactStateLead />
-      </Container>
-    </Html>
+    <BaseEmailTemplate
+      previewText={previewText}
+      heading={heading}
+      applicationEndpointUrl={variables.applicationEndpointUrl}
+      footerContent={<ContactStateLead />}
+    >
+      <DetailsHeading />
+      <PackageDetails
+        details={{
+          "State or territory": variables.territory,
+          Name: variables.submitterName,
+          "Email Address": variables.submitterEmail,
+          "Medicaid SPA ID": variables.id,
+          "Proposed Effective Date": getDateFromMillis(variables.proposedEffectiveDate),
+          "90th Day Deadline": formatNinetyDaysDate(variables.timestamp),
+          Summary: variables.additionalInformation,
+        }}
+      />
+      <Text style={styles.text.base}>
+        {`This response confirms the receipt of your Medicaid State Plan Amendment
+        (SPA or your response to a SPA Request for Additional Information
+        (RAI). You can expect a formal response to your submittal to be issued
+        within 90 days, before ${formatNinetyDaysDate(variables.timestamp)}.`}
+      </Text>
+      <MailboxNotice type="SPA" />
+    </BaseEmailTemplate>
   );
 };
 
-// To preview with 'email-dev'
```

Comment:
Big fan of removing these comments. I don't feel like they add any contextual info. But it should be consistent in the very least